### PR TITLE
[ASM] - EXPANDR-6920 - Active directory changes

### PR DESCRIPTION
**Pull request by capanw**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-dc.paloaltonetworks.com/browse/EXPANDR-6920

## Description
This change is for Active Directory enrichment changes.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Active_Directory_Enrichment.yml [hunk 1]
```diff
@@ -0,0 +1,451 @@
...
+    timertriggers: []
+    type: title
+    view: |-
+      {
+        "position": {
+          "x": 500,
+          "y": 535
+        }
+      }
+  "11":
```
Comment (BigEasyJ) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Active_Directory_Enrichment.yml:205]: I think we should avoid the branching to set values to `asmserviceownerunrankedraw` if possible. Would something like this work for each value:

image

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Enrichment.yml [hunk 2]
```diff
@@ -1235,322 +1253,377 @@ tasks:
...
     view: |-
       {
         "position": {
           "x": 460,
           "y": 2605
         }
       }
-    note: false
-    timertriggers: []
+  "91":
```
Comment (BigEasyJ) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Enrichment.yml:1586]: Please add a header for `Prisma Enrichment` and `Directory Enrichment` above the two task that perform this.

File: Packs/CortexAttackSurfaceManagement/README.md [hunk 3]
```diff
@@ -86,6 +87,7 @@ The main active response playbook is the `Cortex ASM - ASM Alert` playbook. This
   - [Cortex ASM - Prisma Cloud Enrichment](#cortex-asm---prisma-cloud-enrichment)
   - [Cortex ASM - Qualys Enrichment](#cortex-asm---qualys-enrichment)
   - [Cortex ASM - Rapid7 Enrichment](#cortex-asm---rapid7-enrichment)
+  - [Cortex ASM - Active Directory Enrichment](#cortex-asm---active-directory-enrichment)
```
Comment (BigEasyJ) [inline Packs/CortexAttackSurfaceManagement/README.md:90]: Please put in alphabetical order.

File: Packs/CortexAttackSurfaceManagement/README.md [hunk 4]
```diff
@@ -60,6 +60,7 @@ Automated remediation is only possible when the right conditions are met.  These
   - ServiceNow ITSM
   - Tenable.io Assets
   - Qualys
+  - Active Directory
```
Comment (BigEasyJ) [inline Packs/CortexAttackSurfaceManagement/README.md:63]: Please put in alphabetical order.

File: Packs/CortexAttackSurfaceManagement/README.md [hunk 5]
```diff
@@ -191,6 +193,12 @@ A playbook that given the IP address enriches Rapid7 information relevant to ASM
 
 ![Cortex ASM - Rapid7 Enrichment](https://raw.githubusercontent.com/demisto/content/master/Packs/CortexAttackSurfaceManagement/doc_files/Cortex_ASM_-_Rapid7_Enrichment.png)
 
+#### Cortex ASM - Active Directory Enrichment
```
Comment (BigEasyJ) [inline Packs/CortexAttackSurfaceManagement/README.md:196]: Please put in alphabetical order.

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_21.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
+#### Integrations
+##### Cortex Attack Surface Management
+- Updated the Docker image to: *demisto/python3:3.10.13.84405*.
+
+#### Playbooks
+
+##### New: Cortex ASM - Active Directory Enrichment
+
+- Added the **Cortex ASM - Active Directory Enrichment** playbook to enrich service ownership.
```
Comment (BigEasyJ) [inline Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_21.md:9]:
```suggestion
Added the **Cortex ASM - Active Directory Enrichment** playbook to enrich service ownership.
```

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_21.md [hunk 7]
```diff
@@ -0,0 +1,13 @@
+#### Integrations
+##### Cortex Attack Surface Management
+- Updated the Docker image to: *demisto/python3:3.10.13.84405*.
```
Comment (BigEasyJ) [inline Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_21.md:3]:
```suggestion
Updated the Docker image to: *demisto/python3:3.10.13.84405*.
```

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_21.md [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+
+#### Playbooks
+
+##### New: Cortex ASM - Active Directory Enrichment
+
+- Added the **Cortex ASM - Active Directory Enrichment** playbook to enrich service ownership.
+
+##### Cortex ASM - Enrichment
+
+- Updated the playbook to include Active Directory changes in the Enrichment playbook.
```
Comment (BigEasyJ) [inline Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_21.md:13]:
```suggestion
Updated the playbook to include Active Directory changes in the Enrichment playbook.
```

**Review by BigEasyJ — CHANGES_REQUESTED** [review]

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Enrichment.yml (hunk 2 above)
Comment (capanw) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Enrichment.yml:1585]: Added the headers

File: Packs/CortexAttackSurfaceManagement/README.md (hunk 5 above)
Comment (capanw) [reply Packs/CortexAttackSurfaceManagement/README.md]: Done

File: Packs/CortexAttackSurfaceManagement/README.md (hunk 4 above)
Comment (capanw) [reply Packs/CortexAttackSurfaceManagement/README.md]: Done

File: Packs/CortexAttackSurfaceManagement/README.md (hunk 3 above)
Comment (capanw) [reply Packs/CortexAttackSurfaceManagement/README.md]: Done

File: Packs/CortexAttackSurfaceManagement/pack_metadata.json [hunk 9]
```diff
@@ -80,6 +80,10 @@
         "GCP-IAM": {
             "mandatory": false,
             "display_name": "GCP-IAM"
+        },
+        "MicrosoftGraphUser": {
+            "mandatory": false,
+            "display_name": "Microsoft Graph User"
```
Comment (BigEasyJ) [inline Packs/CortexAttackSurfaceManagement/pack_metadata.json:86]: Was this required for validation?

**Review by BigEasyJ — APPROVED** [review]
Great job! Thank you for all of the updates!

File: Packs/CortexAttackSurfaceManagement/pack_metadata.json (hunk 9 above)
Comment (capanw) [reply Packs/CortexAttackSurfaceManagement/pack_metadata.json:86]: Yeah. It is throwing a dependency validation error if we don't add.

**Comment by capanw** [comment]
@BigEasyJ Thank you for reviewing and approving.

Hello @yucohen, this is approved from our side. Please review and merge. Thank you.

**Review by yucohen — APPROVED** [review]

**Pull request merged**